cdap-unit-test-standalone/src/main/java/co/cask/cdap/test/standalone/StandaloneTestBase.java

```diff
@@ -76,4 +89,15 @@ public static void tearDownClass() throws Exception {
       standaloneMain = null;
     }
   }
+
+  protected ClientConfig getClientConfig() {
```

Comment:
For most clientTestRuns, wouldn't we now have two clientConfigs? (one in ClientTestBase, and one here)?